src/components/component-playground.less

```diff
@@ -264,24 +236,86 @@
                            @checkerboard-square-size @checkerboard-square-size;
     }
 
-    &.with-editor {
-      &.orientation-landscape {
-        > .fixture-editor-outer {
-          right: 50%;
-        }
+    > .split-pane {
 
-        > .preview {
-          left: 50%;
+      .fixture-editor-outer {
+        height: 100%;
+        width: 100%;
+        border: 1px solid @editor-border-color;
+        background: #f8f5ec;
+
+        > .fixture-editor {
+          width: 100%;
+          height: 100%;
+          padding: @default-spacing;
+          box-sizing: border-box;
+          border: none;
+          background: transparent;
+          color: #637c84;
+          font-family: @editor-font-family;
+          font-size: @editor-font-size;
+          line-height: @editor-line-height;
+          resize: none;
+          outline: none;
+
+          &.invalid-syntax {
+            color: #cc7a6f;
+          }
         }
       }
 
-      &.orientation-portrait {
-        > .fixture-editor-outer {
-          bottom: 50%;
+      .preview {
+        height: 100%;
+        overflow: auto;
+        // Checkerboard effect on background
+        background-image:
+          linear-gradient(45deg,
+                          @checkerboard-color 25%,
+                          transparent 25%,
+                          transparent 75%,
+                          @checkerboard-color 75%,
+                          @checkerboard-color 100%),
+          linear-gradient(45deg,
+                          @checkerboard-color 25%,
+                          transparent 25%,
+                          transparent 75%,
+                          @checkerboard-color 75%,
+                          @checkerboard-color 100%);
+        background-size: 2 * @checkerboard-square-size
+                         2 * @checkerboard-square-size;
+        background-position: 0 0,
+                             @checkerboard-square-size @checkerboard-square-size;
+      }
+
+      .resizer {
+        background: @resizer-color;
+        z-index: 1;
+        background-clip: padding-box;
+        border: solid transparent;
+
+        &.horizontal {
+          height: @resizer-size;
+          border-width: @resizer-border-size 0;
+          margin: -@resizer-border-size 0;
```

Comment:
Because we need the resizer borders to overlap the two panes. Otherwise it looks bad.